mips/exc.S

```diff
@@ -94,6 +94,67 @@ user_exc_enter:
         addi    t0, 1
         sw      t0, TD_IDNEST(s0)
 
+
```

Comment:
This change is visually distinct from the surrounding code. Please align formatting!